sessions_core/01_introduction.qmd

```diff
@@ -0,0 +1,751 @@
+---
+title: Introduction to R
+description: Your first steps in R. Learn your way around Rstudio, and meet some common R objects.
+date: last-modified
+# image: 
+params: 
+  solutions: FALSE
+  sub_title: "**------ With Solutions ------**"
+categories:
+  - Core
+  - Rstudio interface
+  - R basics
+  - Data types
+---
+
+
+```{r setup}
+#| include: false
+#| eval: true
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+
+solutions <- params$solutions
+```
+
+
+## Objectives
+
+-   Familiarize yourself with RStudio
+-   Learn how to work with the console
+-   Create and execute a script
+-   Create basic R objects, including vectors and dataframes
+
+
+## Exercise Format
+
+These exercises are in the format of a self-paced tutorial containing short explanations of key concepts, examples, and exercises for you to follow. The course uses a “learning by doing” approach, and while this first session will start a slow, giving you time to explore the RStudio interface, future sessions will focus heavily on having you write your own code.
+
+Instructions for exercices will be given in the following formats: 
+
+::: {.setup}
+This is a **general action block**. You will typically see it at the beginning of a session with instructions about the setup for that lesson.
+
+
+**Example:** Open a blank new script and name it `my_first_script.R`.
+:::
+
+
+::: {.write}
+This is a **code block**, it indicates a coding exercise where you will actually write your own code.
+
+
+**Example:** Create an object called `region` that contains the value `"Mandoul"`.
+:::
+
+
+::: {.look}
+This is an **observation block**, it will have instructions about something that you are expected to look at or investigate. 
+
+
+**Example:** Inspect the RStudio interface.
+:::
+
+As you move through these exercises, you may run into some errors.[Errors occur when R is unable to complete a command. This can happen for a lot of reasons, for example because you mispelled the name of an object: you've asked R to look for a data file that doesn't exist, or you've provided the wrong type of data to a function. Whenever an error occurs, R will stop any ongoing calculation and give you a message explaining what went wrong. We will see and discuss a lot of different errors throughout this course.]{.aside} This is completely normal and happens to **all programmers**, novice and expert. Much like a natural language, R is something you will get better at the more you practice and work through your mistakes.
```

Comment:
i'm not fully sold on asides; it isn't always clear to me where they link into the the text and it's not clear what the line is between an aside and a tooltip. for example, i'm not sure why the definition of documentation is an aside.